.github/workflows/no-exception-workflow.yml

```diff
@@ -99,3 +99,22 @@ jobs:
       run: mvn -e --no-transfer-progress clean install -Pno-validations
 
     - run: ./.ci/no-exception-test.sh no-exception-only-javadoc
+
+  no-exception-openjdk14:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up JDK 8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
+
+    - name: Install dependencies
+      run: sudo apt install groovy
+
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Install checkstyle
+      run: mvn -e --no-transfer-progress clean install -Pno-validations
+
+    - run: .ci/no-exception-test.sh openjdk14-with-checks-nonjavadoc-error
```

Comment:
Moved this from Semaphore, it runs much faster with Github action.

Reply:
@nmancus1 , we are very close to completion of migration of pitest to github actions and moving all other runs in circleci. 
Please stay in Semaphore for now.

Reply:
@romani execution is failing in semaphore: https://checkstyle.semaphoreci.com/jobs/8f3e2222-7703-47eb-887c-e487f9095812 . This is due to greatly increased memory usage by ANTLR4 on gigantic repos like openjdk (~50,000 files). Please advise.

Reply:
circle has a low timeout (13 minutes): https://app.circleci.com/pipelines/github/checkstyle/checkstyle/8268/workflows/65554b90-6f92-46fb-bd4f-2b8bdae873be/jobs/97168

Reply:
OOM on circle: https://app.circleci.com/pipelines/github/checkstyle/checkstyle/8324/workflows/f0257a5c-3daf-4265-b154-f3973b7d2304/jobs/97519

Reply:
I am moving this execution back to github action for now.